scripts/tests/scripts_regression_tests.py

```diff
@@ -1463,6 +1467,33 @@ def test_cime_case(self):
             self.assertEqual(case.get_value("TESTCASE"), "TESTRUNPASS")
 
     ###########################################################################
+    def test_cime_case_prereq(self):
+    ###########################################################################
+        testcase_name = 'prereq_test'
+        testdir = os.path.join(TEST_ROOT, testcase_name)
+        if os.path.exists(testdir):
+            shutil.rmtree(testdir)
+        run_cmd_assert_result(self, ("%s/create_newcase --case %s --script-root %s --compset X --res f19_g16 --output-root %s"
+                                     % (SCRIPT_DIR, testcase_name, testdir, testdir)),
+                              from_dir=SCRIPT_DIR)
+
+        with Case(testdir, read_only=False) as case:
+            job_name = "case.run"
+            prereq_name = 'prereq_test'
+            batch_cmd = case.submit_jobs(prereq=prereq_name, job=job_name, skip_pnl=True, dry_run=True)[0][1].split()
+            jobid_ident = 'jobid'
+            dep_str = case.get_env('batch').get_value('depend_string', subgroup=None)[:-len(jobid_ident)]
+            found_dep_str = False
```

Comment:
Lines 1483 and 1485 have a lot going on in them and assume a certain format of the return values, can you
document here what the expected values of batch_cmd and dep_str are?

Reply:
Good point. I added tests for what these should be